bin/init.sh

```diff
@@ -232,6 +232,7 @@ for plugin in stats metadata_exchange
 do
   FILTER_WASM_URL="${ISTIO_ENVOY_BASE_URL}/${plugin}-${ISTIO_ENVOY_VERSION}.wasm"
   download_wasm_if_necessary "${FILTER_WASM_URL}" "${WASM_RELEASE_DIR}"/"${plugin//_/-}"-filter.wasm
+  cp -f "${WASM_RELEASE_DIR}"/"${plugin//_/-}"-filter.wasm "${ISTIO_OUT}"/"${plugin//_/-}"-filter.wasm
```

Comment:
So this change is going to copy these files from `istio.io/istio/out/linux_amd64/release` to `istio.io/istio/out/linux_amd64`? Why? The proxy make file reference $ISTIO_ENVOY_LINUX_RELEASE_DIR which is `istio.io/istio/out/linux_amd64/release`.

Reply:
Hm maybe this isn't quite right. Something is not working for me, but this doesn't seem like the right fix. I'll play around with it some more